oss_lib/config.py

```diff
@@ -0,0 +1,235 @@
+# All Rights Reserved.
+#
+#    Licensed under the Apache License, Version 2.0 (the "License"); you may
+#    not use this file except in compliance with the License. You may obtain
+#    a copy of the License at
+#
+#         http://www.apache.org/licenses/LICENSE-2.0
+#
+#    Unless required by applicable law or agreed to in writing, software
+#    distributed under the License is distributed on an "AS IS" BASIS, WITHOUT
+#    WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied. See the
+#    License for the specific language governing permissions and limitations
+#    under the License.
+
+import argparse
+import copy
+import io
+import logging
+import logging.config
+import os
+import re
+
+import jsonschema
+import six
+import yaml
+
+LOG = logging.getLogger(__name__)
+
+LOG_DATE_FORMAT = "%Y-%m-%d %H:%M:%S"
+LOG_FORMAT = '%(asctime)s - %(name)s - %(levelname)s - %(message)s'
+
+_CONF = None
```

Comment:
nit: we can set it to `{}` to avoid TypeErrors, when accessing uninitialised configs via CONF variable